Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Integral.lean

```diff
@@ -37,111 +37,423 @@ section unital
 
 variable {X : Type*} {𝕜 : Type*} {A : Type*} {p : A → Prop} [RCLike 𝕜]
   [MeasurableSpace X] {μ : Measure X}
-  [NormedRing A] [StarRing A] [NormedAlgebra 𝕜 A] [NormedAlgebra ℝ A] [CompleteSpace A]
+  [NormedRing A] [StarRing A] [NormedAlgebra 𝕜 A] [CompleteSpace A]
   [ContinuousFunctionalCalculus 𝕜 A p]
 
-lemma cfcL_integral (a : A) (f : X → C(spectrum 𝕜 a, 𝕜)) (hf₁ : Integrable f μ)
+lemma cfcL_integral [NormedAlgebra ℝ A] (a : A) (f : X → C(spectrum 𝕜 a, 𝕜)) (hf₁ : Integrable f μ)
     (ha : p a := by cfc_tac) :
     ∫ x, cfcL (a := a) ha (f x) ∂μ = cfcL (a := a) ha (∫ x, f x ∂μ) := by
   rw [ContinuousLinearMap.integral_comp_comm _ hf₁]
 
-lemma cfcHom_integral (a : A) (f : X → C(spectrum 𝕜 a, 𝕜)) (hf₁ : Integrable f μ)
-    (ha : p a := by cfc_tac) :
+lemma cfcHom_integral [NormedAlgebra ℝ A] (a : A) (f : X → C(spectrum 𝕜 a, 𝕜))
+    (hf₁ : Integrable f μ) (ha : p a := by cfc_tac) :
     ∫ x, cfcHom (a := a) ha (f x) ∂μ = cfcHom (a := a) ha (∫ x, f x ∂μ) :=
   cfcL_integral a f hf₁ ha
 
+open ContinuousMap Classical in
+lemma integrable_cfc_set {s : Set X} (hs : MeasurableSet s) (f : X → 𝕜 → 𝕜)
+    (bound : X → ℝ) (a : A) (hf₁ : ∀ x ∈ s, ContinuousOn (f x) (spectrum 𝕜 a))
```

Comment:
Do you want `hf₁` to be an autoparam? I would think so, but maybe you have reasons to avoid it.

Reply:
On second thought, no: `hf₂` refers to `hf₁`, so we can't put it at the end of the list of parameters.